site/content/en/docs/Installation/Install Agones/yaml.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "Install Agones using YAML"
+linkTitle: "YAML"
+weight: 10
+description: >
+  We can install Agones to the cluster using an install.yaml file.
+---
+
+```bash
+kubectl create namespace agones-system
+kubectl apply -f https://raw.githubusercontent.com/googleforgames/agones/{{< release-branch >}}/install/yaml/install.yaml
+```
+
+You can also find the install.yaml in the latest `agones-install` zip from the [releases](https://github.com/googleforgames/agones/releases) archive.
+
+{{< alert title="Warning" color="warning">}}
```

Comment:
Please move this alert to the top. Maybe also explicitly call out that this isn't recommended for production (due to cert reuse)?

Reply:
Tweaked the wording a little, PTAL.